craft/base/crafts.lua

```diff
@@ -341,7 +363,8 @@ function Craft:loadDialog(dialog, user)
             end
         end
     end
-
+    local repairListId = listId
```

Comment:
I may be missing something, but it looks like repairListId is always identical to listId. Am I missing a line, or can this variable be safely dropped and listId used exclusively instead?